duckduckgo/macos-browser
# Improvements in CPM coverage (autoconsent 3.0.0)

Task/Issue URL: https://app.asana.com/0/72649045549333/1203008847079865/f
Product Review: https://app.asana.com/0/1142021229838617/1202892370714490/f
Tech Design URL:
CC:

Description:

This update adds support for a bunch of new CMPs with complex rules and flows, increasing the CPM coverage

Steps to test this PR:

Enable cookie prompt management
Browse some sites from the list in https://app.asana.com/0/1142021229838617/1202892370714490/f
Cookie popups should be rejected automatically
Testing checklist:

  Test with Release configuration
  Test proper deallocation of tabs
  Make sure committed submodule changes are desired
